kuma/javascript/ssr-server.js

```diff
@@ -15,25 +15,10 @@ if (process.env.NEW_RELIC_LICENSE_KEY && process.env.NEW_RELIC_APP_NAME) {
 
 const express = require('express'); // Express server framework
 const morgan = require('morgan');
-const sentry = require('@sentry/node');
+const Sentry = require('@sentry/node');
```

Comment:
All the code examples in the documentation uses this notation for the default export. So I changed to that. Ultimately no functional difference but now our code looks like the docs.